### PR TITLE
SB-1109: configuration resources storage fixed

### PR DESCRIPTION
**Pull request by sbespalov**

### CODE REVIEW

File: strongbox-data-service/src/main/java/org/carlspring/strongbox/data/domain/GenericEntity.java [hunk 1]
```diff
@@ -59,6 +59,7 @@ public final void setObjectId(String objectId)
         this.objectId = objectId;
     }
 
+    @XmlTransient
```
Comment (sbespalov) [inline strongbox-data-service/src/main/java/org/carlspring/strongbox/data/domain/GenericEntity.java]: @fuss86 should it be transient?

Comment (fuss86) [reply strongbox-data-service/src/main/java/org/carlspring/strongbox/data/domain/GenericEntity.java]: Yeah, haven't noticed it before. I think XML file shouldn't have any extra information from our database so `@XmlTransient` would is expected here. Thanks.

Comment (sbespalov) [reply strongbox-data-service/src/main/java/org/carlspring/strongbox/data/domain/GenericEntity.java]: ok
thanks for your response

**Comment by sbespalov** [comment]
@steve-todorov now it's seems to be ready

**Comment by steve-todorov** [comment]
Thanks for the fix @sbespalov! :)

**Pull request merged**